network/connector.go

```diff
@@ -101,6 +101,9 @@ func handleConnecting(log zerolog.Logger, wg *sync.WaitGroup, cfg *Config, pendi
 		return
 	}
 
-	eventMgr.Connected(address)
+	err = eventMgr.Connected(address)
+	if err != nil {
+		log.Debug().Msg(err.Error())
```

Comment:
The logger is used like this:
```
log.Debug().Error(err).Msg("could not submit connected event")
```

Reply:
I don't exactly understand semantic of Debug and then Error, does it mean that we assign two log categories or what?

Reply:
Now I see it, logging error and adding message to it 👍